webdriver-spec.html

```diff
@@ -791,6 +791,11 @@
  has an associated list of <dfn>remote end steps</dfn>.
  This provides the sequence of actions that a <a>remote end</a> takes
  when it receives a particular <a>command</a>.
+ All <dfn data-lt="RES-arguments"><a>remote end steps</a>
+ accept the arguments</dfn> <var>url variables</var> and <var>parameters</var>,
+ obtained from the <a>command</a> as described in the
+ <a href=#processing-model>processing model</a>.
```

Comment:
I think we should explain this when the individual commands are invoked. It’s very hard for us as implementors to parse this out-of-band normative requirement.

If you instead modify step 8 of the processing model to say that the remote end steps takes these variables, it will be much more obvious.